modules/trustxBidAdapter.js

```diff
@@ -313,6 +449,51 @@ function addSegments(name, segName, segments, data, bidConfigName) {
   }
 }
 
+function reformatKeywords(pageKeywords) {
```

Comment:
I see that a bunch of validations is done on the ortb2/fpd object here. Just wanted to let you know that a prebid.js optional modlue [validationFpdModule](https://docs.prebid.org/dev-docs/modules/validationFpdModule.html) was released recently. You may still need this if it's not included in the build.

Do you need to reformat them?